### ISSUE TITLE
Special Support for `.exe` and `.sys` in `maps64`/`maps32`

### ISSUE THREAD

**Issue by sylv256** [sha0coder/libscemu#59]
Currently, `libscemu` tries to find dependencies in `maps64`/`maps32` by adding `.dll` to the end. However, this does not work when the file is a `.exe` or `.sys` (for example, when emulating a `.sys`). The workaround right now is to add `.dll` to the end of these files so that they can be resolved.

**Comment by sha0coder** [comment]
yes i have to refactor this, for now one option is loading it manually:



```
let map = emu.maps.create_map("something")
map.set_base(0x400000)
map.load("file.sys")

in pyscemu:
emu.load_map("code", "memdump_052D0000.bin", 0x52d0000)
```
